README.md

```diff
@@ -129,10 +129,11 @@ After you're done with tests, you can stop Splice Machine using `./start-splice-
 
 ## spark-shell
 
-**NOTE**: [spark-2.4.5-bin-hadoop2.7.tgz](https://archive.apache.org/dist/spark/spark-2.4.5/spark-2.4.5-bin-hadoop2.7.tgz) and 
+**NOTE**: [spark-3.0.1-bin-hadoop2.7.tgz](https://archive.apache.org/dist/spark/spark-3.0.1/spark-3.0.1-bin-hadoop2.7.tgz), 
```

Comment:
Please note [3.0.2](https://spark.apache.org/news/spark-3-0-2-released.html) is the latest version.

Reply:
Thanks for the info, will plan to upgrade when the splice engine project upgrades.